### PR TITLE
Felinid/catpeople are now their own human subspecies

### PR DESCRIPTION
**Pull request by silicons**
/datum/species/human/felinid
tail wag helpers all converted to:
/datum/species/can_tail_wag(mob/living/carbon/human)
/datum/species/is_wagging_tail(mob/living/carbon/human)
/datum/species/start_wagging_tail(mob/living/carbon/human)
/datum/species/stop_wagging_tail(mob/living/carbon/human)

To enable cats you must now put in "felinid" in config entry ROUNDSTART_RACES.
:cl:
rscadd: Catpeople are now a subspecies of human. Switch your character's species to "Felinid" to be one.
rscadd: Oh yeah and they show up as felinids on health analyzers.
/:cl:

### CODE REVIEW

**Comment by Mark9013100** [comment]
anime anime Anime anime

**Comment by Spaceinaba** [comment]
about time

**Comment by Borisvanmemes** [comment]
This'll sadly kill giving non-human races cat ears won't it?

**Comment by kevinz000** [comment]
@Borisvanmemes nope, I've done it before.

**Comment by praisenarsie** [comment]
i swear that's the first time i heard someone called catpeople nekos

**Comment by Arianya** [comment]
Nekomata is better then neko as a species name.

**Comment by Borisvanmemes** [comment]
@granpawalton Cat people never have and never will be counted as human under asimov

**Comment by granpawalton** [comment]
@Borisvanmemes some AI tricked me and ive believed it this whole time

**Review by optimumtact — DISMISSED** [review]

**Comment by GuyonBroadway** [comment]
He did because its his fetish.

I say we call them felinids and give them a graphical overhaul.
https://youtu.be/iUuvHPr4BGk

**Comment by Grazyn** [comment]
Neko is correct because their design is obviously inspired by anime catgirls (_nekomimi_, shortened _neko_), this will also hopefully stop any association with the furry fandom

**Comment by Iatots** [comment]
_codewise_ subspecies but _gameplay_ wise a different species right?

**Comment by vuonojenmustaturska** [comment]
For the record, I requested that @kevinz000 does this since the whole mutant bodypart thing is the wrong place to have catgirls when we have so much code that checks whether someone is a catgirl or not. 

@GuyonBroadway there's nothing stopping you from implementing tattoos even before this gets merged

**Comment by optimumtact** [comment]
@Iatots that's always been the case, but it's a matter of server policy, nothing to do with us.

**Review by fludd12 — COMMENTED** [review]
are people seriously downvoting solely because the PR says "catpeople" lmao

**Comment by LetterN** [comment]
something is nya~t right...

**Comment by Militaires** [comment]
i called it oranges, you never believed me...

**Comment by Militaires** [comment]
btw u should turn can_tail_wag to can_wag_tail if u want it to be consistent and not "oh woops this shit is different for some reason gotta recompile ecks dde"

**Comment by obscolene** [comment]
\>writing an entire page of code for catpeople tail wagging

jesus fucking christ kevinz

**Comment by subject217** [comment]
>felinid
Now that's just boring. They should be called Nekomimi in the code.

**Comment by fludd12** [comment]
**Fel**inid.

Not sure how I feel about it actually.
Fitting, though.

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 1]
```diff
@@ -0,0 +1,130 @@
+//Subtype of human
+/datum/species/human/felinid
+	name = "Felinid"
```
Comment (Wilchenx) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm]: image

name them nekos instead, noone on tg roleplays a grimdark felinid

**Comment by PKPenguin321** [comment]
Can't wait for tattoos, r-right guys?

**Comment by Wilchenx** [comment]
the name is shit, find a better one please

**Comment by optimumtact** [comment]
Okay 300 offtopic comments about naming is really unnecessary.

**Pull request merged**